src/makeRobotAccountant.js

```diff
@@ -6,7 +6,15 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(multiplierOne) {
+    count++;
+
+    return (multiplierTwo) => count > 3 && count % 2 === 0
+      ? `Bzzz... Error!`
+      : multiplierOne + multiplierTwo;
+  };
```

Comment:
if you have some conditions use temples
```suggestion
    return (multiplierTwo) => (
      count > 3 && count % 2 === 0
        ? `Bzzz... Error!`
        : multiplierOne + multiplierTwo
      );
  };
```